Tools/GNUMake/comps/gnu.mak

```diff
@@ -147,11 +147,7 @@ ifdef CXXSTD
   endif
   CXXFLAGS += -std=$(CXXSTD)
 else
-  ifeq ($(gcc_major_version),4)
-    CXXFLAGS += -std=c++11
-  else ifeq ($(gcc_major_version),5)
-    CXXFLAGS += -std=c++14
-  endif
+  CXXFLAGS += -std=c++14
```

Comment:
I think this should be inside `ifeq ($(gcc_major_version),5)` so that we only set the option for gcc 5.  For gcc > 5, we use the compiler's default.

Reply:
Good point, done.